over.css

```diff
@@ -0,0 +1,73 @@
+*{
+    box-sizing: border-box;
+}
+body{
+    
+}
+
+#top{
+    text-align: center;
+    margin: auto;
+}
+
+#top img{
+    display: inline-block;
+    float: left;
+    width: 60px;
+    height: 60px;
+}
+
+#top input[type=text]{
+    width: 75px;
+    height: 75px;
+    text-align: center;
+    font-size: 50px;
```

Comment:
I generally try to use em for font sizes. Read this for more info: https://css-tricks.com/why-ems/